README.md

```diff
@@ -83,20 +92,34 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+app.get("/",function(res,req){
+    res.send("read")
+})
+
+app.post("/",function(res,req){
+  res.send("create")
+})
+
+app.put("/:something",function(res,req){
+  res.send("update")
+})
+
+app.delete("/:anything",function(res,req){
+  res.send("delete")
+})
 ```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+  Express does not generate boilerplate code
```

Comment:
it can, though -- https://expressjs.com/en/starter/generator.html; use the word "rules" in your answer